### PR TITLE
CDAP-8806 Suppress the tephra logs in standalone.

### PR DESCRIPTION
**Pull request by sagarkapare**
JIRA: https://issues.cask.co/browse/CDAP-8806

### CODE REVIEW

File: cdap-standalone/src/main/resources/logback.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
   <logger name="org.spark-project" level="WARN"/>
   <logger name="org.apache.hadoop" level="WARN"/>
   <logger name="org.apache.hive" level="WARN"/>
+  <logger name="org.apache.tephra.TransactionManager" level="WARN"/>
```
Comment (sreevatsanraman) [inline cdap-standalone/src/main/resources/logback.xml]: Reference the JIRA as TODO.

**Comment by sreevatsanraman** [comment]
one comment, LGTM otherwise. fix and merge.

File: cdap-standalone/src/main/resources/logback.xml (hunk 1 above)
Comment (sagarkapare) [reply cdap-standalone/src/main/resources/logback.xml]: fixed.

**Comment by sagarkapare** [comment]
RAT passed. Merging.

**Pull request merged**